src/Compilers/Core/MSBuildTask/ValidateBootstrap.cs

```diff
@@ -77,10 +79,38 @@ public override bool Execute()
             // named pipe errors, CPU load causing timeouts, etc ...  Hence flagging a single failure would produce
             // a lot of false positives.  The current value was chosen as a reasonable number for warranting an 
             // investigation.
-            if (s_failedServerConnectionCount > 20)
+            const int maxRejectCount = 5;
```

Comment:
Are we confident in going from 20 => 5 for the non detectable errors?

Reply:
Yes. Now this should only happen where there is basically a named pipe issue in our build. If there are more than 5 on a regular basis we can kick the number back up.